### DISCUSSION TITLE
"not enough and you get some rock that were sitting on in your final blend"

### DISCUSSION THREAD

**Discussion by leppie** [Extraltodeus/multi-subject-render#4] [General]
Was funny when I was reading it, but at least you know exactly what the problem is :D 

30 minutes later...

Well that is what I thought was the problem. 

Blend noise strength plays a big role here too. Too much, and you will just get rocks too.

Took some time, but finally got something nice (I tried for 50...).

![06797-3674733698-misty, raining, feral hogs, intricate, natural lighting, high quality, 4k, backlighting, natural lighting,ray traced, highly det](https://user-images.githubusercontent.com/127666/203857623-54262795-c45b-47ab-a712-1155132d15b9.png)

**Comment by leppie** [comment]
Settings used (would be nice to be able to save settings somewhere).

![image](https://user-images.githubusercontent.com/127666/203857891-00384a84-da3e-4b97-88b4-b7e10b86c0c9.png)

![image](https://user-images.githubusercontent.com/127666/203857861-88b1b116-e525-49cd-99c6-26afdae120d4.png)

**Comment by Extraltodeus** [comment]
Yeah stuff that is more like 2D or painting requires a lower value :)

Nice result by the way!!! :D
